tsup.config.ts

```diff
@@ -0,0 +1,11 @@
+import { defineConfig } from "tsup";
+
+export default defineConfig({
+  entry: ["src/index.ts", "src/core/index.ts", "src/utils/index.ts", "src/handlers/index.ts", "src/types/index.ts"],
+  splitting: false,
```

Comment:
Tree shaking seems useful is that what this is

Reply:
You mean `splitting` or `tree-shaking`?
Splitting is an option only relevant for `ESM`, not `CJS` and is not handled by `esbuild` which is why it's disabled. `treeShaking` is default on.
https://tsup.egoist.dev/#code-splitting
https://tsup.egoist.dev/#tree-shaking